recipes/pot/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "POT" %}
+{% set version = "0.3.1" %}
+{% set sha256 = "1c226e359e81ca08e593012b9022d64041318d0bd7aafcb30185f1c4f02304af" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - cython
+    - numpy x.x
+    - scipy
+    - matplotlib
+  run:
+    - python
+    - numpy x.x
+    - scipy
+    - matplotlib
+
+test:
+  imports:
+    - ot
+  requirements:
```

Comment:
I don't think you need this `requirements` section since the import test will get all of the packages listed under `run` automatically.